packages/send/frontend/src/stores/auth-store.ts

```diff
@@ -128,6 +143,10 @@ export const useAuthStore = defineStore('auth', () => {
    */
   async function checkAuthStatus() {
     try {
+      // Load user from stored auth data, if available.
+      // No-op if we are not in the add-on.
+      await loadUser();
+
```

Comment:
When other parts of the tb.pro add-on need to check for a user, the first thing we do is check for the stored auth cookie.